tests/helper.go

```diff
@@ -264,6 +264,8 @@ func TestMultiReturnEvalExpr(t *testing.T, tt *MultiReturnEvalTestItem) {
 		}
 
 		for k, v := range wants[0].Tags {
+			// FIXME: we are skipping Tags comparison until we fix the tests https://github.com/grafana/carbonapi/issues/65
```

Comment:
Need to be fixed. It will mask overwrite tags without copy-on-write. And produce some bugs. Lookup the failed tests of functions and and copy-onwrite when needed.